src/main/java/twilightforest/enums/StructureWoodVariant.java

```diff
@@ -81,76 +74,35 @@ public String getName() {
     private final Block gate      ;
     private final Block plate     ;
 
-    //TODO: Probably don't need this
     @Nullable
-    public static BlockPlanks.EnumType getTypeFromBlockState(BlockState stateIn) {
-        Block block = stateIn.getBlock();
+    public static StructureWoodVariant getVariantFromBlockState (BlockState state) {
+        Block block = state.getBlock();
+        return getVariantFromBlock(block);
+    }
 
-        if (!"minecraft".equals(block.getRegistryName().getNamespace())) return null;
+    // Redone to make more sense with the above
```

Comment:
will this comment make sense 6 months from now? (redone w.r.t what?)
I'd just remove it.

Reply:
Very true, removed.